site/zh/blog/new-in-devtools-104/index.md

```diff
@@ -1,18 +1,18 @@
----
+---	
 layout: "layouts/blog-post.njk"
 title: "DevTools 新功能（Chrome 104）"
 authors:
   - jecelynyeen
 date: 2022-07-13
 updated: 2022-07-13
-description: "Restart frame during debugging, slow replay options in the Recorder panel, and more."
+description: "在调试时进行帧重启, 录制面板中的慢速复盘选项与更多。"
 hero: 'image/dPDCek3EhZgLQPGtEG3y0fTn4v82/VojSLwN9rFRkALzi6RZh.jpg'
 alt: ''
 tags:
   - new-in-devtools
   - devtools
   - chrome-104
-draft: true
+draft: false
 ---
 
 *感谢 [流浪大法师 @liuliangsir](https://github.com/liuliangsir) 提供的翻译*。
```

Comment:
@xyugroup Please replace it with you name and url (social media, github, linkedin, etc).